app/lib/actions/katello/capsule_content/sync.rb

```diff
@@ -70,7 +70,6 @@ def sync_repos_to_capsule(capsule_content, repository_ids, skip_metadata_check)
                 if skip_metadata_check
                   plan_action(Katello::Repository::MetadataGenerate,
                               repo,
-                              capsule_id: capsule_content.capsule.id,
```

Comment:
Removed this as `Katello::Repository::MetadataGenerate` doesnt seem to used a `capsule_id` option (not related to this change), maybe that's another bug as I think currently a 'forced' smart proxy sync will regenerate metadata on the "master" pulp server?